hypothesis-python/src/hypothesis/internal/conjecture/data.py

```diff
@@ -330,84 +331,97 @@ def freeze(self):
         self.events = frozenset(self.events)
         del self._draw_bytes
 
-    def draw_bits(self, n):
+    def blocks_with_values(self):
+        for b in self.blocks:
+            n = int_from_bytes(self.buffer[b.start : b.end])
+            assert bit_length(n) <= b.bits
+            yield (b, n)
+
+    def draw_bits(self, n, forced=None):
+        """Return an ``n``-bit integer from the underlying source of
+        bytes. If ``forced`` is set to an integer will instead
+        ignore the underlying source and simulate a draw as if it had
+        returned that integer."""
         self.__assert_not_frozen("draw_bits")
         if n == 0:
             result = 0
```

Comment:
This assignment appears to no longer have any effect, since `result` is unconditionally reassigned later on.